modules/nw-sriov-create-object.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// * networking/hardware_networks/using-dpdk-and-rdma.adoc
+
+:_content-type: REFERENCE
+[id="nw-sriov-create-object_{context}"]
+= Example SR-IOV network operator
+
+The following is an example definition of an `sriovNetwork` object. In this case, Intel and Mellanox configurations are identical:
+[source,yaml]
+----
+    apiVersion: sriovnetwork.openshift.io/v1
```

Comment:
I just noticed that this whole example is indented. Should the top level start with no indentation and everything else align from that? Please check with an SME or QE reviewer.